NuRadioReco/modules/channelGalacticNoiseAdder.py

```diff
@@ -8,20 +8,31 @@
 import matplotlib.pyplot as plt
 import scipy.constants
 import scipy.interpolate
-#from radiocalibrationtoolkit import * #Documentation: https://github.com/F-Tomas/radiocalibrationtoolkit/tree/main contains SSM, GMOSS, ULSA
+
+logger = logging.getLogger('channelGalacticNoiseAdder')
```

Comment:
Please use the NRR logger naming scheme: "NuRadioReco.channelGalacticNoiseAdder"